projectDocs/issues/triage.md

```diff
@@ -143,10 +143,10 @@ An issue is triaged if it is ready to be worked on.
 New features and enhancements should be [well defined](#new-features-1) before applying the `triaged` label.
 Once a bug has clear steps to reproduce and is well documented, the `triaged` label can be applied.
 A `triaged` issue should also have a [priority label](#priority).
-Generally these labels are provided by NV Access only, particularly for issues that are controversial, or priority is unclear.
 
-For controversial changes, a product decision from NV Access may be required before applying the `triaged` label.
-This can be indicated with adding the label `blocked/needs-product-decision`.
+Community members should avoid adding a `triaged` label to feature requests or other product decisions, deferring to NV Access.
```

Comment:
@CyrilleB79 @XLTechie How about this?

```suggestion
Community members should avoid adding a `triaged` label to feature requests or other decisions potentially involving significant or controversial changes to NVDA features or functionality, which may require community input or approval from NV Access.
```

Reply:
Should clarify "may require community input" to "may require more extensive community consultation"?

(Just thinking any member of the community reading a given issue and wanting to triage something could arguably be providing "community input")